src/handlers/nus/modules.json

```
@@ -1,4 +1,13 @@
 {
+  "CS2030S": {
+    "luminus": "c8378b00-daa4-430e-b49f-22f6509a1287"
+  },
+  "CS20404S": {
```

Typo?

---

I'm assuming this is CS2040S so I'll just correct it to that so that we can get this in

```suggestion
  "CS2040S": {
```